scripts/qstash-manager.js

```
@@ -0,0 +1,353 @@
+    const schedules = await makeQStashRequest('/schedules');
+
+    // Count schedules by priority (extracted from headers)
+    const priorityCounts = {
+      critical: 0,
+      high: 0,
+      medium: 0,
+      low: 0,
+      unknown: 0,
+    };
+
+    schedules.forEach((schedule) => {
```

ESLint

[error] 'schedule' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)